api/SVGRenderingIntent.json

```diff
@@ -14,10 +14,10 @@
             "version_added": null
           },
           "firefox": {
-            "version_added": null
+            "version_added": false
```

Comment:
From https://developer.mozilla.org/docs/Web/API/SVGRenderingIntent it looks like we should probably remove this entry. But this PR is fine.